src/common/Tile/styled.js

```diff
@@ -0,0 +1,188 @@
+import styled, { css } from "styled-components";
+
+export const StyledTile = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 324px;
+  min-height: 650px;
+  border-radius: 5px;
+  background-color: white;
+  padding: 16px;
+  box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
```

Comment:
Może warto dodać kolor do theme, żeby było spójnie?

Reply:
Pewnie, można tak zrobić, chociaż w pamięci mi zapadło, jak kiedyś tak robiłem i dałem Krzyśkowi do review, to mówił, że to przesada takie podstawowe kolory wrzucać do theme.js, ale jak najbardziej możemy wrzucić ;)

Reply:
Mi się kojarzy z jego zadań domowych, że wszystkie kolory dawał do tego pliku, ale spoko, jak tak mówił przy review to może tak lepiej, nie upieram się :)

Reply:
Może nie trzeba jak to jest podstawowy kolor, np wynikający z body. Ja bym dał do theme.js dla spójności konwencji.

A swoją drogą osobiście uważam, ze taki color theme bazujący na nazwach kolorów to nie jest wcale dobra praktyka. I trochę bezsensowna. Bo jeśli zechcemy zmienić kolor danego elementu to i tak w całym kodzie trzeba będzie to zmienić bo ma inną nazwę.
O wiele bardziej uważam, że lepiej jest nazywać po przeznaczeniu: np. primary, secondary, accent itp. Wtedy jak chcemy żeby główny np nie był czarny a graftiwy to zmieniamy w jednym miejscu.
Ale tutaj nie będziemy tego zmieniać oczywiście, żeby nie robić zamieszania.

Reply:
zgadzam się Tomek co do tego theme, ale jak już przyjmujemy jakąś konwencję to ja osobiście lubię jak jest spójnie. Możemy w naszym projekcie zrobić to inaczej i nie kierować się do końca zaleceniami Chrisa, ale wtedy musimy sobie ustalić jak to ma wyglądać i tej naszej konwencji się trzymać.

Reply:
Nie no jasne, robimy tak jak w kursie żeby dla każdego były znane i jasne konwencje. To tak tylko poza tematem napisałem :)